i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/quick_start.md

```diff
@@ -15,7 +15,7 @@ sidebar_position: 2
   ```
 
 ## 添加依赖
-- 下载 Sort [Connector Binary file](https://inlong.apache.org/download/main), 并解压放入`inlong-sort/connectors`目录。
+- 下载 Sort [Connector Binary file](https://inlong.apache.org/download), 并解压放入`inlong-sort/connectors`目录。
```

Comment:
Here should be https://inlong.apache.org/zh-CN/download/ for Chinese doc.

Reply:
Ok, I have fixed this issue.

Reply:
I have also modified the links to other Chinese versions as well.